### DISCUSSION TITLE
Unlinking from Secondary account

### DISCUSSION THREAD

**Discussion by Sidd-17052004** [community/community#132988] [GitHub Education] [answered]
### Hi everyone,

Hello dear supporters! 👋

Today, I made a mistake, in which I had linked my school email to this GitHub account to get the Student Benefits, but after I linked, I realised that this is my secondary account. Can I transfer the benefits to my principal account, and also remove them from this account?

I have also made same problem. What if I unlink my student mail and add to my primary account. Will my student benefits be there . Please tell fast. It will be very helpful

**Comment by iamdulanga** [comment, marked as answer]
Hello, @Sidd-17052004 

**Actually you can transfer your github student benefits to your primary account, I will tell you how to do it below.**

**1. Remove the email address that you have got the benefits from secondary account.**

- Navigate to emails settings under this account and,
- Remove email from this account.

**2. Add your student email to your primary account:**

- After that, login to your github account that wants to move benefits,
- and go to settings option and go to emails,
- After that add your student email.

**3. Reapply for the Student Developer Pack:**

- It is possible to have problems in this process, you will have to get GitHub support.

Hope this will help to you..
Dulanga.

**Comment by davevad93** [comment]
Hi @Sidd-17052004 ,

It's not possible to transfer Student benefits from one account to another. You can merge your accounts although the account with the benefits would be the main one. For more information on merging accounts, take a look at [Merging multiple personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) from the docs.

For more reference on the topic here's the official response from a member of the GitHub Education team . #49875
